assets/course-theme/blocks/blocks.js

```diff
@@ -180,6 +181,101 @@ const blocks = [
 			);
 		},
 	},
+	{
+		...meta,
+		title: __( 'Quiz Progress Bar', 'sensei-lms' ),
+		name: 'sensei-lms/quiz-progress-bar',
```

Comment:
We don't have a block for the `sensei-lms/quiz-progress-bar`. Also, it wasn't rendered for me in WP 5.9.
Are you planning to add this block? If you didn't have plans, I'll create an issue for that, so we can work on that. ;)

Reply:
Right, yes, we still need to build that one. The existing component also needs changing: https://github.com/Automattic/sensei/issues/4644
Added the block work to that issue and added it to the project board now.